.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - v*.*.*
```

Comment:
In relation to this [comment](https://github.com/elastic/ecctl/pull/123#discussion_r364861327) are we going to keep the `v` prefix?

Reply:
The v prefix is coming from the semver specification. We can keep it or just ignore it and on every tag with `*. *. *` if that's better

Reply:
> just ignore it and on every tag with *.*.* if that's better

This sounds good to me if everyone's on board :)